autotest/gdrivers/ecw.py

```diff
@@ -2210,6 +2210,66 @@ def test_ecw_read_uint32_jpeg2000():
     )
 
 
+###############################################################################
+# Test unsupported XML SRS
+
+
+def test_jp2ecw_unsupported_srs_for_gmljp2(tmp_vsimem):
+
+    if gdaltest.jp2ecw_drv is None or not has_write_support():
+        pytest.skip("ECW write support not available")
+
+    filename = str(tmp_vsimem / "out.jp2")
+    # No EPSG code and Albers Equal Area not supported by OGRSpatialReference::exportToXML()
```

Comment:
?
```suggestion
    # No EPSG code and Albers Equal Area are supported by OGRSpatialReference::exportToXML()
```

Reply:
changed to "    # There is no EPSG code and Albers Equal Area is not supported by OGRSpatialReference::exportToXML()"
done